core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -61,7 +62,7 @@ object ZkSecurityMigrator extends Logging {
                       + "znodes as part of the process of setting up ZooKeeper "
                       + "authentication.")
 
-  def run(args: Array[String]): Unit = {
+  def run(args: Array[String], checkPathExists: Boolean = true): Unit = {
```

Comment:
Normally users use `./bin/zookeeper-security-migration.sh` for migration. Adding method parameter may not be useful. maybe we can enable path check using cmd line option.